wallet/walletrpc.c

```diff
@@ -771,6 +771,12 @@ static struct command_result *json_listfunds(struct command *cmd,
 		list_for_each(&p->channels, c, list) {
 			json_object_start(response, NULL);
 			json_add_node_id(response, "peer_id", &p->id);
+			/* Mirrors logic in listpeers */
+			json_add_bool(response, "connected",
+				      channel_active(channel)
```

Comment:
Should be, `c`, or `c` above should be `channel`

Reply:
... and I should compile-test at least, before pushing!!